pallets/parachain-staking/src/delegation_requests.rs

```diff
@@ -154,7 +154,7 @@ impl<T: Config> Pallet<T> {
 				error: <Error<T>>::DelegatorBondBelowMin.into(),
 			},
 		);
-		let new_amount: BalanceOf<T> = (bonded_amount - decrease_amount).into();
+		let new_amount: BalanceOf<T> = bonded_amount.saturating_sub(decrease_amount);
```

Comment:
this is kinda big deal, I am wondering if there are more instances of using unsafe arithmetic in the staking code